src/utils/clm_time_manager.F90

```diff
@@ -520,16 +527,15 @@ subroutine timemgr_restart( )
 
     dtime = rst_step_sec
 
-    ! Initialize start date from restart info
-
-    start_date = TimeSetymd( rst_start_ymd, rst_start_tod, "start_date" )
+    ! Check start date from restart info
```

Comment:
The timemgr_spmdbcast and init_calendar calls above can also be removed, because this now requires timemgr_init to be called first. As such we should check that

timemgr_set == .true.

and abort if not.

Reply:
Moving this as a thing to do in https://github.com/ESCOMP/CTSM/pull/2912